pkg/operator/ceph/controller/spec.go

```diff
@@ -645,6 +681,19 @@ func (c *daemonConfig) buildAdminSocketCommand() string {
 	return command
 }
 
+func (c *daemonConfig) buildDaemonReadinessCheckCommand() string {
+	command := ""
+	if c.daemonType == config.MgrType {
+		// As per the current ceph mgr implementation, only active manager
+		// offers the "mgr_status" command. Standby interface list this command.
+		command = "mgr_status"
+	} else if c.daemonType == config.MonType {
```

Comment:
AFAIK, the mon daemon does not use the readiness probe, and in the case that we will need the readiness probe for the mon, we do not know is it is going to be based in the output of the admin socket command "mon_status"

Reply:
Fixed. I removed the mon part from all the related code.